CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- DscResource:
+    - `nxScript`: Simple resource for executing scripts.
```

Comment:
nit: Simple resource for executing scripts **in PowerShell Core/PowerShell 7**
(Just want customers to know that we aren't running this as a bash or python or PS5 script)